docker/dev/php/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.0-cli-alpine3.16
+FROM php:8.2-cli-alpine3.16
```

Comment:
Why not the latest version?

Reply:
I only raised it to 8.2, so that composer would install `symfony/console` version 7 as it is the minimum requirement of the package (^8.2), so I can double-check if all is good.

But if you want, I can raise it to latest version?

Reply:
Since we are already updating it, If possible, it would be great to raise it to the latest stable version

Reply:
updated :)